src/command_modules/azure-cli-servicebus/azure/cli/command_modules/servicebus/_help.py

```diff
@@ -568,3 +577,27 @@
         - name:  Disable Service Bus Migration of Standard to Premium namespace
           text: az servicebus migration abort --resource-group myresourcegroup --name standardnamespace
 """
+
+helps['servicebus namespace network-rule add'] = """
+    type: command
+    short-summary: Set a Network rule for a Namespace
+    examples:
+        - name: set a Network rule for a namespace
+          text: az servicebus namespace network-rule add --resource-group myresourcegroup --namespace-name mynamespace
```

Comment:
The description should be "Add a network rule...". Also, this isn't a valid example. Either a subnet or IP address must be added. I would recommend adding an example for each.

Reply:
corrected examples